src/storage/indexed-db-adapter.test.ts

```
@@ -0,0 +1,301 @@
+    });
+
+    it('should not throw error if IndexedDB is available', () => {
+      expect(() => adapter['verifyEnvironment']()).not.toThrow();
+    });
+  });
+
+  describe('initDB', () => {
+    it('should initialize database successfully', async () => {
+      const initPromise = adapter['initDB']();
+      mockIDBRequest.onsuccess?.(new Event('success'));
+      
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)